debian/Dockerfile

```diff
@@ -0,0 +1,36 @@
+FROM debian:stretch
+MAINTAINER Shaun Jackman <sjackman@gmail.com>
+
+# Set apt in non-interactive mode during the image build
+ARG DEBIAN_FRONTEND=noninteractive
+
+# Update debian package list
+RUN apt-get update
```

Comment:
It allows to save an image with an up-to-date list of packages. If you change the list of package to install, the first image don't have to be rebuild (apt-get update is not run again).